devs/docs/basics.rst

```diff
@@ -70,14 +70,28 @@ If you want to attach a debugger to the instance, you'd have to enable the debug
 Then create an "Attach" configuration in your IDE or editor. If using Visual
 Studio Code you can use the pre-defined "Attach to CrateDB" configuration.
 
+Using an IDE
+============
+
+`IntelliJ IDEA`_ will pick up project structure from the maven poms, but you will need to configure
```

Comment:
To import the project into `IntelliJ IDEA`_, first, build the sources outside of your IDE with::

    $ ./mvnw clean install -DskipTests=true

and then import the project by selecting the root `pom.xml`.  `IntelliJ IDEA`_ will pick up project structure from the maven poms, but you might need to configure the 'crate-libs-sql-parser' generated sources manually, due to an `IDE bug`_.  Go to Project Settings / Modules / crate-libs-sql-parser and mark '>target/generated-sources/antlr' as a Source Folder.

Finally, run a `git clean` to remove pointless IntelliJ file changes to checked in config files.

Reply:
There are btw. more generated sources. `pgwire` also uses antlr for the PostgreSQL text format array parser and `server` uses it for type signatures.